### PR TITLE
src: fix src-lite

### PR DESCRIPTION
**Pull request by lyakh**
This is a "more gentle" attempt to fix src-lite than #9182

### CODE REVIEW

**Review by lgirdwood — APPROVED** [review]

**Comment by lgirdwood** [comment]
@singalsu pls review

File: tools/rimage/config/mtl.toml.h [hunk 1]
```diff
@@ -116,10 +116,6 @@
 #include <audio/codec/dts/dts.toml>
 #endif
 
-#ifdef CONFIG_COMP_SRC_LITE
-#include <audio/src/src_lite.toml>
-#endif
-
 #ifdef CONFIG_CADENCE_CODEC
 #include <audio/module_adapter/module/cadence.toml>
 #endif
```
Comment (singalsu) [inline tools/rimage/config/mtl.toml.h:121]: Why no tgl.toml and tgl-h.toml update?

**Review by singalsu — APPROVED** [review]
Approve, because the code looks right. The issues in running src_lite do not seem to be caused by this PR. Normal src works.

But I wasn't able to run this in (IPC3) testbench despite doing the testbench changes to load the component and created the tplg1 test pipelines for it. The component load in IPC3 didn't pass correctly the source and sink rate, so prepare() failed. IPC3 for src_lite needs more work. It likely needs another IPC exception add to module adapter to pass the larger than normal process IPC. But src_lite was never intended for IPC3 so is probably not needed.

With IPC4 and real TGL UPX device I did the the tgl.toml edit to be able to load the component with topology sof-hda-benchmark-src_lite32.tplg. However I'm not able to play anything non-48 kHz with desired rate. The topology or kernel may have an issue in formats with rates.

**Comment by singalsu** [comment]
@lyakh With fix to benchmark topologies #9218 the PCM now accepts non-48 kHz. The muted audio with src_lite impacts only 48000 to 48000 playback. Apparently the pass-through is not working in your version. The playback with 32000 or 44100 works normally with sound heard.

File: tools/rimage/config/mtl.toml.h (hunk 1 above)
Comment (lyakh) [reply tools/rimage/config/mtl.toml.h:121]: tgl doesn't do modular toml

**Comment by lyakh** [comment]
> How did you test this PR?

@singalsu I only tested pipelines with full src (without checking the audio quality) and ran testbench. I couldn't test src-lite with real audio for 2 reasons: (1) we don't have such topologies at least not among those, used with the CI and (2) I think, that src-lite is currently broken in the sense, that even if you test such a topology, what you test won't actually be src-lite but the normal full-scale src, because src-lite coefficients are never enabled.
I propose a choice: either we remove src-lite completely because it's currently broken and unused, or we take "any" fix that makes it usable "formally" and if any audio performance isn't correct with it, it should be fixed incrementally.

**Review by singalsu — APPROVED** [review]
I just discussed with Guennadi. The src_lite is still broken with this PR. But it does not seem to be due to these changes, probably has been broken in the original code. We can approve this and file a bug and decide later what to do with this.

**Review by kv2019i — APPROVED** [review]

**Pull request merged**